NtupleProducer/test/step1_L1TCorr_EGNtuples.py

```diff
@@ -183,7 +174,7 @@
 # process.L1simulation_step.remove(process.L1TkIsoElectronsHGC)
 
 # Schedule definition
-# process.schedule = cms.Schedule(process.raw2digi_step,process.L1TrackTrigger_step,process.L1simulation_step,process.ntuple_step)
+process.schedule = cms.Schedule(process.raw2digi_step,process.L1TrackTrigger_step,process.L1simulation_step,process.ntuple_step,process.endjob_step,)
```

Comment:
Do you really need to schedule all steps explicitly? The previous version was meant to pull tasks needed by the ntuplizer with some options to avoid re-running track triggers or some other components.
Was it not working?

Reply:
Maybe not in fact. I uncommented that line after I had issues initially and did not try to go back when I had fixed them.

Reply:
Can we maybe try to address those problems before reverting to a fully scheduled approach? Not a big deal but being able to quickly switch on/off parts of the processing can speed up things. Especially if we eventually manage to define an input skim to streamline the ntuple production.
Thanks!

Reply:
Should be reverted now.